hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/SharedNetworkLauncherSessionListener.java

```diff
@@ -245,6 +246,53 @@ private static void startSharedEmbedded(@NonNull final EmbeddedMode mode) {
             SHARED_NETWORK.get().start();
         }
 
+        private static void upsertApplicationOverride(
```

Comment:
Note more to myself than anything, one of the reasons the hapi framework is so hard to use is because of all the (undifferentiated) overrides, specifically a lack of clarity about which override applies in which context 🤔